### PR TITLE
[W02] Add logging tests

### PR DESCRIPTION
**Pull request by ITegs**
#### General
- [x] I locally pass **all** tests or calculated manually the correct solutions on paper.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://github.com/JohannesStoehr/gad23-tests/blob/main/CONTRIBUTING.md).
- [x] The newly added tests do not reveal the solutions dynamically calculating the solutions as expected in the exercise.

### Description
Tests the logging output of all search types
Largely inspired by tests from @Flnn0 and @antecrescent 🙌🏼 

#### Review
- [ ] Review 1
- [ ] Optional Review 2

### CODE REVIEW

File: w02/test/gad/logging/TestResultImplementation.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package gad.logging;
+
+import gad.Result;
```
Comment (Anatoly03) [inline w02/test/gad/logging/TestResultImplementation.java:3]:
```suggestion
import gad.binarysearch.Result;
```

File: w02/test/gad/logging/loggingTest.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
+package gad.logging;
+
+import gad.BinSea;
+import gad.Interval;
```
Comment (Anatoly03) [inline w02/test/gad/logging/loggingTest.java:4]:
```suggestion
import gad.binarysearch.BinSea;
import gad.binarysearch.Interval;
```

**Review by Anatoly03 — CHANGES_REQUESTED** [review]
fix import path

**Comment by Anatoly03** [comment]
I pass all tests, lgtm other than the import issue.

File: w02/test/gad/logging/TestResultImplementation.java (hunk 1 above)
Comment (ITegs) [reply w02/test/gad/logging/TestResultImplementation.java]: fixed with 150c24dff1e115fe23e718d4a850c977a5ded1e1

File: w02/test/gad/logging/loggingTest.java (hunk 2 above)
Comment (ITegs) [reply w02/test/gad/logging/loggingTest.java]: fixed with 150c24dff1e115fe23e718d4a850c977a5ded1e1

**Pull request merged**